deploy/helm/sumologic/conf/logs/logs.source.containers.conf

```diff
@@ -25,7 +25,7 @@
   #  only match fluentd logs based on fluentd container name. by default, this is
   #  <filter **collection-sumologic**>
 {{ if .Values.fluentd.logLevelFilter }}
-  {{ printf "<filter **%s**>" (include "sumologic.fullname" .) }}
+  {{ printf "<filter **%s-fluentd**>" (include "sumologic.fullname" .) }}
```

Comment:
we should use helper here instead of concatenating file manually

Reply:
👍 helpers are here: https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/helm/sumologic/templates/_helpers.tpl#L46

Reply:
I actually changed several templates to make it consistent. Let me know what do you think @sumo-drosiek @perk-sumo